apps/namadillo/src/App/Sidebars/ValidatorDiversification.tsx

```diff
@@ -19,6 +19,7 @@ export const ValidatorDiversification = (): JSX.Element => {
         textHoverColor="black"
         backgroundHoverColor="cyan"
         href="https://namada.net"
```

Comment:
do you know if we have a better link for this? It looks like just a "lorem ipsum"

Reply:
As far as I know, we don't. I was thinking of reviewing all the links together and making a list for Awa or Chris to decide on, which is why I didn't touch the link in this PR. I'll open a new issue for it.

Reply:
https://github.com/anoma/namada-interface/issues/985

Reply:
I found this now :) 
https://namada.net/blog/namada-cubic-proof-of-stake